site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -746,7 +746,12 @@ components:
                type: array
                items:
                   type: string
-               description: The device IDs of the gateways this device is assigned to.
+               description: The device IDs of the gateways and the IDs of the gateway groups that are registered to act on behalf of this device. Note that "via" and "memberOf" must not be set at the same time.
```

Comment:
FMPOV the ambiguous usage of this field to contain both device IDs as well as group IDs will make the process of resolving the list of gateways IDs unnecessarily complicated. IMHO we should use a dedicated property for gateway groups instead, e.g. *viaGroups*. This would also make it easier to take advantage of referential integrity checks if the persistence technology supports it. I also have a hard time imagining how a UI should process this mixed list of gateway and group IDs which need to resolved differently ...

@ctron @dejanb @jbtrystram @calohmn any opinion on this?

Reply:
Yes, I agree, a separate _viaGroups_ property would make things easier and more explicit here.

Reply:
I agree too and will add the _viaGroups_ property.